packages/components/QuantityField.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  InputGroup,
+  InputLeftAddon,
+  Input,
+  InputRightAddon,
+} from '@chakra-ui/react';
+interface Props {
+  value: number;
+  onChange: (value: number) => void;
+}
+export default function QuantityField(props: Props) {
+  const handleClick = () => {
+    if (props.value - 1 < 1) {
```

Comment:
podemos simplificar o condicional. 
Outro questão é que não é uma boa prática deixar números soltos no código como esse `1`. Podemos transformá-lo em uma constante para deixar o código mais expressivo.
(a constante pode ser definida fora do QuantityField: `const MIN_QTY = 1`)

```suggestion
    if (props.value <= MIN_QTY) {
```